lib/schemaUtils.js

```diff
@@ -521,6 +521,7 @@ module.exports = {
    */
   generateTrieFromPaths: function (spec, options, fromWebhooks = false) {
     options = _.merge({}, defaultOptions, options);
+    spec = deref.dereferenceByConstraint(spec, /#\/components\/pathItems/);
```

Comment:
We added a dereferencing from paths here in the generateTrieFromPaths to homologate the process we were following of dereferencing elements from '#/components' when we will use the value.

Reply:
@erfemega I meant to add comment in code itself. This way we can get contest while working.